serverless.yml

```diff
@@ -0,0 +1,125 @@
+service: JPC-Messaging-Service
+
+plugins:
+  - serverless-python-requirements
+
+custom:
+  serviceName: JPC-Messaging-Service
+  bucketName: 'cunningham-message-bucket'
+  shared_message_bus: ${ssm:/${self:provider.stage}/serverless/tbg_message_bus}
+  pythonRequirements:
+    pythonBin: python3
+    dockerizePip: true
+
+package:
+  patterns:
+    - "!.venv/"
+    - "!node_modules/"
+
+provider:
+  name: aws
+  runtime: python3.8
+  stage: ${opt:stage, 'dev'}
+  iamRoleStatements:
+    - Effect: Allow
+      Action:
+        - s3:*
+      Resource:
+        - 'arn:aws:s3:::${self:custom.bucketName}'
+        - 'arn:aws:s3:::${self:custom.bucketName}/*'
+    - Effect: Allow
+      Action:
+        - sqs:SendMessage
+        - sqs:SendMessageBatch
+        - sqs:ReceiveMessage
+        - sqs:ReceiveMessageBatch
+        - sqs:DeleteMessage
+        - sqs:DeleteMessageBatch
+        - sqs:GetQueueUrl
+      Resource:
+        - arn:aws:sqs:*:*:${self:custom.CunninghamQueue}
+
+  environment:
+    S3_BUCKET: ${self:custom.bucketName}
+    CUNNINGHAM_QUEUE_URL: { Ref: CunninghamQueue }
+
+# function for logging the event
+functions:
+  MessageLogger:
+    handler: app.handler
+    Environment:
+      Variables:
+        LOG_LEVEL: INFO
+        POWERTOOLS_SERVICE_NAME: ${self.custom.ServiceName}
```

Comment:
You have this as an environment variable, but you're not using it in line 3 of your app.py